lib/nvmf/rdma.c

```diff
@@ -52,8 +52,6 @@
 #include "spdk/nvmf_spec.h"
 #include "spdk/string.h"
 #include "spdk/trace.h"
-#include "spdk/util.h"
```

Comment:
Seems unrelated, and this file uses definitions from util.h (e.g. spdk_min), so it shouldn't be removed.